firefox-code-search/background.js

```diff
@@ -0,0 +1,83 @@
+const SRC_URL = "http://searchfox.org/mozilla-central";
+const PATH_PREFIX = "path:";
+
+chrome.omnibox.setDefaultSuggestion({
+  description: "Search the firefox codebase (e.g. `hello world` | `path:omnibox.js hello world`"
+});
+
+chrome.omnibox.onInputEntered.addListener((text, disposition) => {
+  let url = text;
+  if (!text.startsWith(SRC_URL)) {
+    url = `${SRC_URL}/search?q=${text}+-path%3Aobj-&redirect=false`
+  }
+  chrome.tabs.query({active: true, currentWindow: true}, tabs => {
+    chrome.tabs.update(tabs[0].id, {url});
+  });
+});
+
+function buildURL(text) {
+  let path = '';
+  let queryParts = [];
+  let query = '';
+  let parts = text.split(' ');
+
+  parts.forEach((part, i) => {
+    if (part.startsWith(PATH_PREFIX)) {
+      path = part.slice(PATH_PREFIX.length);
+    } else {
+      queryParts.push(part);
+    }
+  });
+
+  query = queryParts.join(' ');
+  return `${SRC_URL}/search?q=${query}&case=false&regexp=${isRegex}&path=${path}`;
```

Comment:
When testing this I get `isRegex is not defined` and I can't spot it anywhere else in here.